### PR TITLE
refactor: Add `addClass` and `removeClass` methods to blockSvg

### LINKED ISSUE #8270: Add `addClass` and `removeClass` methods to the block to deal with CSS classes
## To Fix

1. Add an `addClass` method to the BlockSvg class, that takes in a passed string, and calls `dom.addClass` with the `this.svgGroup_` to add it as a CSS class on the block.
2. Add a `removeClass` method to the BlockSvg class, that takes in a passed string, and calls `dom.removeClassClass` with the `this.svgGroup_`.
3. Refactor any calls in the BlockSvg class to `dom.addClass` with `this.svgGroup_` to call `this.addClass` instead.
3. Refactor any calls in the BlockSvg class to `dom.removeClass` with `this.svgGroup_` to call `this.removeClass` instead.

The PR to fix this should be made against the `rc/v12.0.0` branch.

To work on this issue, please comment below asking to be assigned.

### PR DESCRIPTION
**Pull request by ishon19**
## The basics

- [x] I [validated my changes](https://developers.google.com/blockly/guides/contribute/core#making_and_verifying_a_change)

## The details
This PR add `addClass` and `removeClass` methods, which accept the class name as a parameter and can be used to perform the `dom.addClass` call rather than calling it directly.

Fixes #8270

### CODE REVIEW

File: core/block_svg.ts
Comment (BeksOmega) [inline core/block_svg.ts:675]:
```suggestion
   * Add a CSS class to the SVG group of this block.
```

For extra clarity :P

File: core/block_svg.ts
Comment (BeksOmega) [inline core/block_svg.ts:678]:
```suggestion
```

We actually want these to be public! You can remove the `@internal` tags from `addClass` and `removeClass`.

File: core/block_svg.ts
Comment (BeksOmega) [inline core/block_svg.ts:688]:
```suggestion
```

**Review by BeksOmega — CHANGES_REQUESTED** [review]
Thanks for the fix! Just some docs nits. Once these are fixed I'll pull your changes down and give them a test.

File: core/block_svg.ts
Comment (ishon19) [reply core/block_svg.ts]: That makes sense! Updated.

File: core/block_svg.ts
Comment (ishon19) [reply core/block_svg.ts]: Absolutely! Updated.

**Review by BeksOmega — APPROVED** [review]
This LGTM! Once CI passes I'll get this merged. Thanks for digging into this =)

**Comment by ishon19** [comment]
> This LGTM! Once CI passes I'll get this merged. Thanks for digging into this =)

Thanks @BeksOmega!

**Pull request merged**
